### DISCUSSION TITLE
Heroic Games Launcher will not open on Steam Deck

### DISCUSSION THREAD

**Discussion by novasheikh** [Heroic-Games-Launcher/HeroicGamesLauncher#1416] [Q&A] [answered]
I am a Linux noob with a steam deck here. Sorry if my question is basic. I have encountered a bug where heroic games launcher refuses to open. It installs ok and in system monitor I see it running but the actual window for the application never appears. I have Uninstalled and reinstalled it several times. I even went to the github to download it directly but weirdly enough that github installation never officially registered on my device in the menu (it was the app image version and I can't figure out how to uninstall that version). I wonder if it is possible that the github installation remains on my device and is preventing the installation from discover from working correctly.

For what it is worth, things at least launched for me with heroic before I tried installing the github app image. I appreciate any advice or help!

Update- I didn't realize I should have posted this in the Issues section. This is the first time I have ever used Github discussions/issues so if this discussion needs to be deleted I understand as I recreated this post under Issues. Thanks!

**Comment by novasheikh** [comment, marked as answer]
I ended up fixing my issue by deleting the legendary.exe file as I had accidentally set Heroic Games Launcher to use this Windows version of legendary

**Reply by Rashimotosan** [reply]
Can you explain where that file is? I'm having same issue. Thank you

**Reply by novasheikh** [reply]
> Can you explain where that file is? I'm having same issue. Thank you

I had downloaded it into the downloads folder. You should be able to search for it. I went through a very specific set of steps to generate my issue that I am not sure may apply to you though. I now have a better understanding of Linux and how things work in KDE desktop. I had downloaded the windows version of Legendary (legendary.exe) and manually specified that Heroic should use this specific executable. Of course that means Heroic Games Launcher is going to say that it is missing some Windows components since it is being told to use the wrong Legendary version.

If you did not manually specify Heroic to use the windows version of legendary then you should not have as difficult a time as me. If you did, just search specifically for legendary.exe and delete it (assuming you manually downloaded it yourself accidentally). After that, Heroic should default back to the linux version of legendary that comes with the flatpak you installed on the discover store.

See this link for more details:
https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/issues/1417
